core/bubbles/mini_workspace_bubble.ts

```diff
@@ -196,6 +229,20 @@ export class MiniWorkspaceBubble extends Bubble {
     return new Size(width, height);
   }
 
+  /** Reapplies styles to all of the blocks in the mini workspace. */
+  updateBlockStyles() {
```

Comment:
If we update the styles of blocks in the main workspace, we want to update the styles of blocks in the mini workspace bubble as well. This update method is called by the icon, since the main workspace itself has no knowledge of mini workspace bubbles.